ts/RailroadMap.ts

```diff
@@ -143,10 +146,15 @@ export class RailroadMap {
     private remainingTreesAppender?: (trees: Vector[]) => Promise<void>;
     private readonly mergeLimits: MergeLimits;
 
+    // required for later beta and release maps of RRO
+    // older maps should still be 180 degrees rotated to match the in-game map
+    private legacyRotate = false;
+
     constructor(studio: Studio, element: HTMLElement) {
         this.setMapModified = (affectsSplines = false) => studio.setMapModified(affectsSplines);
         this.setTitle = (title) => studio.setTitle(title);
         this.railroad = studio.railroad;
+        this.legacyRotate = this.isLegacyRotationSaveGame();
```

Comment:
I recommend moving the `isLegacyRotationSaveGame` function and the related constant to `Studio.ts`, as we already doing some save game version analysis in there. You can pass `inverted` to the map's constructor by adding an argument here:

```suggestion
    constructor(studio: Studio, element: HTMLElement, inverted: boolean) {
        this.setMapModified = (affectsSplines = false) => studio.setMapModified(affectsSplines);
        this.setTitle = (title) => studio.setTitle(title);
        this.railroad = studio.railroad;
        this.inverted = inverted;
```